### PR TITLE
Remove Google Fonts

### PR DESCRIPTION
**Pull request by qwerty287**
Serve fonts directly from local files.
Closes https://github.com/LycheeOrg/Lychee-front/issues/208

### CODE REVIEW

**Comment by ildyria** [comment]
I disagree with this last commit, if you enable the Landing page, roboto is being used.

**Comment by ildyria** [comment]
> Do you mean we shoud keep Roboto there? Because I changed the font used on the landing page, it is not longer Roboto (take a look at CSS changes in this PR).

Yes I saw and tested. :)

With Roboto:
2022-05-06_1920x1080_11:50:57
2022-05-06_1920x1080_11:52:52

Without:
2022-05-06_1920x1080_11:52:56
2022-05-06_1920x1080_11:53:31

It is a personal opinion, but I quite preferred the thin look that Roboto gives compared to depending on the random Sans-serif font used by the browser (to be clear I am not opposed to hearing other opinions).

**Comment by kamil4** [comment]
@ildyria I can certainly understand your preference in this case. However, the question is: _Should this be the default?_ In particular, if this effect can be achieved using our custom CSS feature (I haven't tried so I don't know), wouldn't that be the way to go for advanced users, rather than pushing non-standard fonts on everybody?

If the use of the custom CSS is problematic in this case, I'm certainly OK with using a (self-hosted) Roboto font on the landing page though, as an initial version of this PR did...

**Comment by nagmat84** [comment]
What is the progress of this PR? I believe all arguments have been said. @qwerty287 has committed a user guidance on how to create a custom `user.css` which re-enables the Roboto font.

I feel rather indifferent about the decision. But I would like to see this PR merged soon, either as it is now (with no dependency on Roboto at all) or as it was before the last commit (with a self-hosted version of Roboto). How do we come to a decision?

**Comment by nagmat84** [comment]
We could also take a ballot among the active developers. As we are five (@ildyria, @kamil4, @d7415, @qwerty287 and me) we will get a definitive decision, if no one abstains.

File: styles/landing/landing.scss [hunk 1]
```diff
@@ -1,5 +1,28 @@
 // Rest -------------------------------------------------------------------- //
-@import url("https://fonts.googleapis.com/css?family=Roboto:300,400,700,900");
+/* roboto-300 - vietnamese_latin-ext_latin_greek-ext_greek_cyrillic-ext_cyrillic */
+@font-face {
+	font-family: "Roboto";
+	font-style: normal;
+	font-weight: 300;
+	src: local(""), url("../../../fonts/roboto-v29-300.woff2") format("woff2");
```
Comment (kamil4) [inline styles/landing/landing.scss:7]: Just curious -- why does this need `"../../../"`? What directory are we in when these need to be loaded? I can see that it's correct from the point of view of the Lychee front end sources, but once those are compiled, doesn't that whole nested folder structure get forgotten?

**Review by kamil4 — APPROVED** [review]
Approved, assuming it works (see my question about the relative paths).

**Pull request merged**